.github/workflows/create-rc-release.yaml

```
@@ -0,0 +1,77 @@
+        with:
+          go-version: "1.23.0"
+
+      - name: Add rocket emoji to comment
+        run: |
+          curl -X POST -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+          -d '{"body": "🚀 Starting the release process!"}' \
+          "https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/comments"
+
+      - name: Parse release version from PR title
+        id: parse_release_version
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)